packages/components/file-card/src/states/download.js

```diff
@@ -0,0 +1,37 @@
+import { calcFileSize } from '../utils/calcFilesize.js';
+
+export function downloadState(fileInfo, viewOnly) {
+  const fileExtension = fileInfo.name.split('.').pop();
+  const closeIcon = `
+    <div class="file-card--close icon">
+      <span class="file-card--close">x</span>
+    </div>
+  `;
+  const downloadIcon = `
+    <div class="file-card--downlaod icon">
+      <span class="file-card--downlaod">↓</span>
+    </div>
+  `;
+
+  return `
+  <div class="file-card file-card--download">
+    <div class="file-info--wrapper">
+      <div class="icon-wrapper">
+        <div class="icon">
+          <svg xmlns="http://www.w3.org/2000/svg" width="35" height="44" viewBox="3 1 18 22" fill="none" stroke="currentColor" stroke-width="1" stroke-linecap="round" stroke-linejoin="round" class="feather feather-file">
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Split this 220 characters long line (which is greater than 120 authorized). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.ts.sv/coding_rules#rule_key=javascript%3ALineLength)